src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -21,23 +21,26 @@
 
 (def row-height-px 56)
 
-(def disabled-text
-  (str "This workspace contains controlled access data. You can only access the contents of this workspace if you are "
-       "dbGaP authorized for TCGA data and have linked your FireCloud account to your eRA Commons account."))
+(defn- disabled-text [protected?]
```

Comment:
I don't like having a noun as the name of a method, could this be a verb somehow?